### ISSUE TITLE
Requesting help from Windows/Linux to smoke test jar file

### ISSUE THREAD

**Issue by natlv** [nus-cs2103-AY2425S1/forum#413]
Looking for help in smoke-testing my IP. The user guide is linked [here](https://natlv.github.io/ip/) and the release JAR is [here](https://github.com/natlv/ip/releases/tag/A-Release). 

Thank you and I appreciate your help.

**Comment by starbucksventi** [comment]
I got the following error when running your code. Might have to do with your build.gradle file, if you're using a Mac (i'm a Windows user)
![image](https://github.com/user-attachments/assets/5015359a-80d2-4022-8bec-a4ff179ffc96)

![image](https://github.com/user-attachments/assets/b1dbd191-e038-4ae4-9b38-57a92c52ff2e)

Edit: Your build.gradle seems to be missing several dependencies. Try to add the dependencies from here: https://se-education.org/guides/tutorials/javaFxPart1.html
![image](https://github.com/user-attachments/assets/07b67a6f-68d2-4e02-a974-6ad4eae08e06)

**Comment by flyingsalsa** [comment]
![image](https://github.com/user-attachments/assets/ac2e3465-555a-426f-a335-f6cef72d4082)
Same as above 
you need to bundle the decencies into your build.gradle 
check if you included all the dependencies for all platforms from the [javaFX tutorial](https://se-education.org/guides/tutorials/javaFxPart1.html)

**Comment by natlv** [comment]
I've re-updated my dependencies as shown in the screenshot below. The updated jar release is [here](https://github.com/natlv/ip/releases/tag/A-Release). Please let me know if the issue persists. Thank you.

<img width="856" alt="Screenshot 2024-09-23 at 7 59 50 AM" src="https://github.com/user-attachments/assets/254a7c4b-f9c8-40eb-a21f-cdf6155a20df">

**Comment by LimKaiWei** [comment]
Unfortunately the same issue still persists! does the system work on your end? try 
```
./gradlew clean shadowjar
```
(might be related #411)

**Comment by minhao23** [comment]
![image](https://github.com/user-attachments/assets/e360c03f-68a4-473b-8246-56b6a2f3f615)
seems to be an issue with dependencies

**Comment by natlv** [comment]
Hi, I believe I managed to fix the issue. Those who are still interested to help test may try [here](https://github.com/natlv/ip/releases/tag/A-Release).

It seemed the original problem was with javafx.application.Application being in the original Launcher class. After refactoring my Launcher class to not include Application and then rebuilding with `./gradlew clean shadowjar`, the problem appears to have been resolved.
